src/libs/ascent/runtimes/flow_filters/ascent_runtime_conduit_to_vtkm_parsing.cpp

```diff
@@ -191,6 +196,124 @@ parse_camera(const conduit::Node camera_node, vtkm::rendering::Camera &camera)
       vtkm::Float64 elevation = camera_node["elevation"].to_float64();
       camera.Elevation(elevation);
   }
+
+  // ----------------------- //
+  // Visit camera parameters //
+  // ----------------------- //
+  if(camera_node.has_child("focus"))
+  {
+      conduit::Node n;
+      camera_node["focus"].to_float64_array(n);
+      const float64 *coords = n.as_float64_ptr();
+      vtkmVec3f look_at(coords[0], coords[1], coords[2]);
+      camera.SetLookAt(look_at);
```

Comment:
Is there a reason for this style? You could do this more simply like this:
```suggestion
      float64_accessor coords = camera_node["focus"].value();
      // data_accessors give you type-coerced data access.
      vtkmVec3f look_at(coords[0], coords[1], coords[2]);
      camera.SetLookAt(look_at);
```
You could use this for all of the cases to avoid making copies of all of the arguments.